jax/_src/numpy/polynomial.py

```diff
@@ -102,15 +114,156 @@ def roots(p: ArrayLike, *, strip_zeros: bool = True) -> Array:
     return _roots_with_zeros(p_arr, num_leading_zeros)
 
 
-_POLYFIT_DOC = """\
-Unlike NumPy's implementation of polyfit, :py:func:`jax.numpy.polyfit` will not warn on rank reduction, which indicates an ill conditioned matrix
-Also, it works best on rcond <= 10e-3 values.
-"""
-@implements(np.polyfit, lax_description=_POLYFIT_DOC)
 @partial(jit, static_argnames=('deg', 'rcond', 'full', 'cov'))
 def polyfit(x: Array, y: Array, deg: int, rcond: float | None = None,
             full: bool = False, w: Array | None = None, cov: bool = False
             ) -> Array | tuple[Array, ...]:
+  r"""Least squares polynomial fit that returns the polynomial coefficients of
```

Comment:
If the one-line description doesn't fit on one line, it's too long. Perhaps something like this instead:
```
Least squares polynomial fit to data
```

Reply:
Done